arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'How to run an orbit node'
+description: Learn how to run an Arbitrum orbit node on your local machine.
+sme: Mehdi Salehi
+sidebar_label: Run an orbit node
+sidebar_position: 12
+content-type: how-to
+---
+
+This guide provides a step-by-step process for running an Orbit node on your local machine. To initiate an Orbit node, the `"chain info"` json string of the orbit chain, which includes crucial configurations required for the Orbit chain's operation. The `"chain info"` json string should be provided by the orbit chain owner, and the node runner can consume it to run the node for that specific orbit chain. An example of the "chain info" json string is available in this section. 
+
+
+### Minimum hardware configuration
+
+- Followings specify the minimum hardware configuration required to setup a Nitro full node (not archival):
+  - RAM: 8-16 GB
+  - CPU: 2-4 core CPU (For AWS: t3 xLarge)
+  - Storage: It depends on the orbit chain and its traffic during the time. 
+
+### Prerequisites
+
+- Latest Docker Image: <code>@latestNitroNodeImage@</code>
+
+### Required parameters
+
+- Parent chain RPC URL
+  - Use the parameter `--parent-chain.connection.url=<Parent chain RPC URL>`
+  - It must provide a standard RPC endpoint for an EVM node, whether self-hosted or obtained from a node service provider
+    - Would be set to appropriate Arbitrum One, Arbitrum Nova, or Arbitrum Sepolia RPC URL
+  - Note: [Arbitrum RPC endpoints](/node-running/node-providers.mdx/#rpc-endpoints) RPCs rate limit connections, So you will need to run a local node for the parent chain or rely on third party RPC providers
+- L3 orbit chain info and chain name
+  - Providing the flag --chain.info-json=<Orbit Chain's chain info> is mandatory. The chain info json string contains vital information about the Orbit chain, which is necessary for the node to commence operations. An example of the chain info json string is available in the subsequent sections.
+  - `--chain.name=<My Arbitrum L3 Chain>` must be provided, and it must match the chain name that is used in `--chain.info-json` content.
+  - For an Anytrust chain, it is essential to provide `--node.data-availability.enable`.
```

Comment:
Do full nodes need this too?

Reply:
Yeah all of them needs this now, but in future version, Nitro will send an update so that just having info for the chain would be enough